### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by schuermannator**
`CONTRIBUTING.md` and `GetInvolved` component

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Contributing to STEV
+
+Thank you for considering making contributions to the project! A brief outline
```
Comment (samjett247) [inline CONTRIBUTING.md]: 2nd Sentence -> This document contains a brief outline of the guidelines for contributing.

File: src/components/GetInvolved.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <h3>Current Contribution Suggestions</h3>
+      <p><i>Updated Spring 2019</i></p>
+      <ul>
+        <li><b>Bug Tracking:</b> Submit a bug you discovered! Or submit an issue regarding the lack of data. If you discover a bug, make sure to jot down the steps to reproduce and submit a bug report in the "Issues" tab on our <a href="https://github.com/stev-ou">GitHub</a>. The latter contribution is especially valuable since our team has attempted to work with the university many times over, with very little receptiveness or action on the university's behalf. As we collect more information about missing data, and the demand for rectifying it, we will continue our attempts to gather more data from the university.</li>
+        <li><b>Suggesting Improvements:</b> After playing with the application, we encourage you to submit an issue on our <a href="https://github.com/stev-ou">GitHub</a> with the tag "Feature Request", where developers will be able to discuss and hopefully implement your request!</li>
```
Comment (samjett247) [inline src/components/GetInvolved.js]: Change "tag" to "label" I think since thats what its called on Github

File: src/components/GetInvolved.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      <p>Thanks for your interest in contributing to STEV! As this project continues to grow, there are increasingly many opportunities for involvement. Those with engineering backgrounds are encouraged to jump into the code and submit a pull request (more on that <a href="#pull-requests">below</a>). No matter your background, there are a number of ways to contribute to the project. A few recent challenges are listed here. </p>
+      <h3>Current Contribution Suggestions</h3>
+      <p><i>Updated Spring 2019</i></p>
+      <ul>
+        <li><b>Bug Tracking:</b> Submit a bug you discovered! Or submit an issue regarding the lack of data. If you discover a bug, make sure to jot down the steps to reproduce and submit a bug report in the "Issues" tab on our <a href="https://github.com/stev-ou">GitHub</a>. The latter contribution is especially valuable since our team has attempted to work with the university many times over, with very little receptiveness or action on the university's behalf. As we collect more information about missing data, and the demand for rectifying it, we will continue our attempts to gather more data from the university.</li>
```
Comment (samjett247) [inline src/components/GetInvolved.js]: "As we collect more information about the demand for missing evaluations, we will continue ..." in last sentence

Comment (samjett247) [inline src/components/GetInvolved.js]: Idk i think these first three sentences could be revised to be more clear? just so it flows better from topic to topic.

File: src/components/GetInvolved.js (hunk 2 above)
Comment (samjett247) [reply src/components/GetInvolved.js]: Change "feature request" to "enhancement" unless we want to make a new label for feature request.

File: src/components/GetInvolved.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <li><b>Suggesting Improvements:</b> After playing with the application, we encourage you to submit an issue on our <a href="https://github.com/stev-ou">GitHub</a> with the tag "Feature Request", where developers will be able to discuss and hopefully implement your request!</li>
+        <li><b>Source Code Contributions:</b> All source code is available online, hosted on <a href="https://github.com/stev-ou">GitHub</a>. We also use GitHub for bug tracking, issues, and suggested improvements as stated above. For example, browse the issues for the frontend repository <a href="https://github.com/stev-ou/stev/issues">here</a>. For those with software development experience, we would love your help. Our current team is small and we graciously welcome source contributions! For more information, see "Pull Requests" below.</li>
```
Comment (samjett247) [inline src/components/GetInvolved.js]: Change sentence 3,4 to "For those interested in contributing to software development, we encourage you to browse the *issues* in the frontend repository [here]() and find an issue you would like to address"

File: src/components/GetInvolved.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      </ul>
+      <h3 id="pull-requests">Pull Requests</h3>
+      <p>Our source code is hosted on GitHub, in the organization <a href="https://github.com/stev-ou">stev-ou</a>. In order to make contributions, first fork and clone our repository to your local machine and build the site. The build instructions are included in the <a href="https://github.com/stev-ou/stev/blob/master/README.md">README</a> of each repository. Please consult the <a href="https://github.com/stev-ou/stev/blob/master/CONTRIBUTING.md">CONTRIBUTING.md</a> file for actionable details regarding contributing to our source. After creating a branch including your features/bug fixes, open a pull request for review by our team. After passing the review and our CI/CD pipeline, your contributions will be included in our source! If there are a number of contributions from an individual, we will happily welcome you as a maintainer of the project if interested.</p>
```
Comment (samjett247) [inline src/components/GetInvolved.js]: Change "After creating a branch including your features/bug fixes" to "After creating a new branch and implementing your desired changes"

Comment (samjett247) [inline src/components/GetInvolved.js]: Change the last sentence to "If you make many relevant contributions to the STEV source code, we would be happy to welcome you as a project administrator."

File: src/components/GetInvolved.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      <h3>Contact</h3>
+      Reach our team for questions at <a href="mailto:contact@evals.info">contact@evals.info</a>.
```
Comment (samjett247) [inline src/components/GetInvolved.js]: Just a suggestion - Change to "For questions or to get in touch, reach out to our team at ...".

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+follows, in order to provide a brief foundation for contributions. If you have
+any questions, feel free to drop us a line at
+[contact@evals.info](mailto:contact@evals.info).
+
+
+#### Table of Contents
```
[the remark this answers is not in the document]
Comment (schuermannator) [reply CONTRIBUTING.md]: did ###

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  * [Getting Started](#getting-started)
+  * [Design Decisions](#design-decisions)
+
+## Bug Reporting
+Our bug reports are handled via GitHub issues. Upon discovering a bug, open an
+issue with the tag "bug" and provide the steps to reproduce. 
+
+## Suggesting Improvements
+Similar to bug reports, suggesting improvements or new features are also handled
+via GitHub. In order to file a suggestion/enhancement, open an issue in GitHub
```
[the remark this answers is not in the document]
Comment (schuermannator) [reply CONTRIBUTING.md]: linked

File: src/components/GetInvolved.js [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+<li><b> :</b> Poll for student opinion in an effort to gather feedback and observe usage patterns. Take Student Name, ID and an "I approve" box?Captcha?Enforce 1 submission/IP?</li>
+*/
+
+const GetInvolved = props => (
+  <div className="container">
+    <div className="dataview-container">
+      <h1 style={{marginBottom: ".75em"}}><b>Get Involved</b></h1>
+      <p>Thanks for your interest in contributing to STEV! As this project continues to grow, there are increasingly many opportunities for involvement. Those with engineering backgrounds are encouraged to jump into the code and submit a pull request (more on that <a href="#pull-requests">below</a>). No matter your background, there are a number of ways to contribute to the project. A few recent challenges are listed here. </p>
```
[the remark this answers is not in the document]
Comment (schuermannator) [reply src/components/GetInvolved.js]: refactored

File: src/components/GetInvolved.js (hunk 2 above)
Comment (schuermannator) [reply src/components/GetInvolved.js]: 👍

File: src/components/GetInvolved.js (hunk 6 above)
Comment (schuermannator) [reply src/components/GetInvolved.js]: changed to match About

**Pull request merged**